main.py

```diff
@@ -178,7 +178,19 @@ async def compile_project(project_root: Path, manifest: PackageManifest):
     # Create a contracts directory
     contracts_dir = project_root / "contracts"
     contracts_dir.mkdir()
-    project = Project.from_manifest(manifest)
+
+    # Hacked in for testing..
+    project = Project.from_manifest(
+        manifest,
+        config_override={
+            "dependencies": [
+                {
+                    "pypi": "snekmate",
+                    "config_override": {"base_path": "src", "contracts_folder": "snekmate"},
```

Comment:
So one goal of mine is to get Ape able to detect this config from projects like snekmate.
I am not 100% sure on how to do that yet. For now, may have to hardcode it in the hosted-compiler

Reply:
Great goal for us, but keep in mind that likely remix may have a set of whitelisted dependencies here w.r.t. snekmate (maybe not whitelist the version, but how the object is formed otherwise)